### ISSUE TITLE
Feature Request: Side-by-side translation panel in visual editor

### ISSUE THREAD

**Issue by DBailey635** [Orckestra/C1-CMS-Foundation#450]
When updating the translation of a web-page in the secondary language site, it would be useful to have a panel next to the visual editor that shows the original language version of the page, so that you can check your translation as you go.

The positioning of this panel would need to take account of screen aspect ratio, so if viewed on a tablet device in portrait mode, the panel would appear above the visual editor... and if in landscape mode, to the right of the visual editor.

**Comment by DBailey635** [comment]
Thinking further, if screen space is at a premium, it may be beneficial to add a button to minimise the page tree (close the drawer to the left) when translation mode is active.

**Comment by burningice2866** [comment]
@DBailey635 are you thinking in terms of a preview of the page itself or a secondary Visual Editor with the content of the page in the original language?

**Comment by DBailey635** [comment]
@burningice2866 No, not the whole page with all of it's chrome. I'd envisage just the text of the original page would appear, as if viewed in Visual Editor, but non-editable.

The only editable area would be the secondary language page. I'll try and produce a screenshot to illustrate.

**Comment by DBailey635** [comment]
@burningice2866 Screenshot:

![translation mode](https://user-images.githubusercontent.com/14888727/26975801-3aad107c-4d19-11e7-8c2c-ecf60b62647f.png)

**Comment by burningice2866** [comment]
@DBailey635 What if the content consist of a single Function which then in turn take other functions as parameters which finally have some text :) And what about the placeholders when a template consist of two or more placeholders, you seem to have left them out of the screenshot.

**Comment by DBailey635** [comment]
It would have to use the PhantomJS preview engine. Indeed, as the original text would be non-editable, using Phantom would be ideal.
